### PR TITLE
Privacy policy: Add section for video conferences

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

File: content/de/privacy.md [hunk 1]
```diff
@@ -162,6 +162,15 @@ Die Anmeldung ist vollkommen freiwillig für Dich.
 - Weitergabe der Daten: Auf die Kontaktdaten hat nur der Vorstand Zugriff. Der Newsletter wird über den Dienstleister Mailjet verschickt. Dafür werden die Kontaktdaten an Mailjet übertragen.  
   Mailjet wird betrieben von der {{< link slug="company/mailjet" text="Mailjet SAS" >}}, 13-13 bis rue de l’Aubrac, 75012 Paris, Frankreich. Mehr Informationen dazu, wie Mailjet Deine Daten verarbeitet, kannst Du deren [Datenschutzerklärung](https://www.mailjet.com/privacy-policy/) entnehmen.
 
+#### Videokonferenzen
+
+Wir verwenden ein Videokonferenzsystem, etwa für öffentliche Veranstaltungen und Mitgliederversammlungen. Im Rahmen von Videokonferenzen ist die flüchtige Verarbeitung einiger Daten nötig.
+
+- Betroffene Daten: Anzeigename/Pseudonym, ggf. Audio- und Videodaten (Mikrofon, Webcam, Bildschirm), hochgeladene Präsentationen/Bilder, Chatnachrichten, geteilte Notizen, Abstimmungsergebnisse, Raumname der Konferenz, technische Metadaten (Beginn und Ende der Konferenz, Anzahl der Teilnehmenden), Telefonnummer (nur bei Telefoneinwahl)
```
Comment (zner0L) [inline content/de/privacy.md:169]: What about public IP address and (potentially) leaked local IP?

Comment (baltpeter) [reply content/de/privacy.md:169]: I left these out because they should already be covered by "Server-Verbindungsdaten", shouldn't they?

Comment (zner0L) [reply content/de/privacy.md:169]: But in this case we send them to another provider.

Comment (baltpeter) [reply content/de/privacy.md:169]: Eh, I don't care.

**Review by zner0L — APPROVED** [review]

**Pull request merged**